### PR TITLE
    Split standard_crt build step into separate script to not build standard_crt on hexagon

### PR DESCRIPTION
**Pull request by alanmacd**
Split standard_crt build step into separate script to not build standard_crt on hexagon.

### CODE REVIEW

File: ci/jenkins/Test.groovy.j2 [hunk 1]
```diff
@@ -88,8 +92,8 @@
   num_shards=8,
 ) %}
   {{ m.download_artifacts(tag='hexagon', filenames=tvm_lib, folders=hexagon_api) }}
-  add_hexagon_permissions()
   ci_setup(ci_hexagon)
+  add_hexagon_permissions()
```
Comment (alanmacd) [inline ci/jenkins/Test.groovy.j2:96]: need to fix this

**Review by mehrdadh — COMMENTED** [review]
LGTM! I'll wait for @driazati to take a look

**Comment by driazati** [comment]
Code looks good to me, if it works I’m good with merging but #13335 does this along with some other cleanups. To test we’ll need to push the PR branch to apache/tvm to actually run the Jenkinsfile changes

**Comment by driazati** [comment]
Can we close this since #13335 is merged?

**Pull request closed without merging**